src/util/tooltipLabel.js

```diff
@@ -0,0 +1,49 @@
+export function getReactEventLabel(type) {
+  let label = null;
+  switch (type) {
+    case 'schedule-render':
+      label = 'render scheduled';
+      break;
+    case 'schedule-state-update':
+      label = 'state update scheduled';
+      break;
+    case 'schedule-force-update':
+      label = 'force update scheduled';
+      break;
+    case 'suspense-suspend':
+      label = 'suspended';
+      break;
+    case 'suspense-resolved':
+      label = 'suspense resolved';
+      break;
+    case 'suspense-rejected':
+      label = 'suspense rejected';
+      break;
+    default:
+      break;
+  }
+  return label;
+}
+export function getReactMeasureLabel(type) {
```

Comment:
style nit: there should be an empty line between functions

```suggestion

export function getReactMeasureLabel(type) {
```